Dockerfile

```diff
@@ -5,6 +5,7 @@ COPY . /src
 
 RUN mkdir /out
 RUN rustup component add rustfmt
+RUN cargo update
```

Comment:
We should not run `cargo update` here.
This should be done locally and the cargo lock file should reflect which dependencies we want to install